src/observable/GenerateObservable.ts

```diff
@@ -245,7 +245,7 @@ export class GenerateObservable<T, S> extends Observable<T> {
     } while (true);
   }
 
-  private static dispatch<T, S>(state: SchedulerState<T, S>) {
+  private static dispatch<T, S>(state: SchedulerState<T, S>): Subscription | void {
```

Comment:
@kwonoj 

I thought it's bit broader for this. This `dispatch` will not return `Function` ever.